packages/mgt-chat/src/components/ChatListHeader/EllipsisMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
I am not strongly opinionated on this, but Menu already is a re-usable component so this would just be wrapping a bit more. The ellipsis-menu is already used in several places (without a re-usable component), so it would mean replacing those as well.